Pipfile

```diff
@@ -8,7 +8,7 @@ verify_ssl = true
 mock = "*"
 pytest-cov = "*"
 jsonschema = "*"
-pylint = "*"
+pylint = "2.17.7"
```

Comment:
Isn't the issue with version 3?. If so, the pin should be set to less than version 3.

Reply:
updated